ts/src/bingx.ts

```diff
@@ -2161,7 +2161,7 @@ export default class bingx extends Exchange {
                 positionSide = (side === 'buy') ? 'LONG' : 'SHORT';
             }
             request['positionSide'] = positionSide;
-            request['quantity'] = this.parseToNumeric (this.amountToPrecision (symbol, amount));
```

Comment:
@Dan-krm this is causing failure, shouldn't we format the quantity here?

Reply:
@Dan-krm Yes that happens when no precision is available, and yes you can separate the logic to avoid impacting the other markets

Reply:
done